.ci/install_podman.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#
+# Copyright (c) 2029 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+cidir=$(dirname "$0")
+default_kata_config="/usr/share/defaults/kata-containers/configuration.toml"
+kata_config="/etc/kata-containers/configuration.toml"
+
+setup_prerequisites() {
+	# Verify host kernel version
+	host_kernel_version=$(uname -r|cut -d. -f1-2)
+	result=$(echo "${host_kernel_version} >= 4.14"|bc)
+	[ "${result}" -ne 1 ] && echo "ERROR: host kernel version too old" && exit 1
+
+	# Disable selinux
+	if  [ "$(getenforce)" != "Disabled" ]; then
+		sudo -E setenforce 0
+		sudo -E sed -i 's/^SELINUX=enforcing/SELINUX=disabled/g' /etc/sysconfig/selinux
+	fi
+
+	# Add user to KVM group
+	getent group kvm &>/dev/null || sudo groupadd --system kvm
+	sudo usermod -a -G kvm $USER
+	newgrp kvm
+
+	# Ensure /dev/kvm device is group owned
+	sudo chown root:kvm /dev/kvm
+	sudo chmod g+rw /dev/kvm
+}
+
+setup_kata_configuration_files() {
+	sudo mkdir -p $(dirname "${kata_config}")
+	[ ! -e "${kata_config}" ] && sudo install -D "${default_kata_config}" $(dirname "${kata_config}")
+
+	sudo chown root:kvm "${kata_config}"
+	sudo chmod g+r "${kata_config}"
+}
+
+disable_vhost_net() {
+	sudo sed -i -e 's/^#disable_vhost_net = true/disable_vhost_net = true/' "${kata_config}"
+}
+
+modify_kata_image_permissions() {
+	TRUSTED_GROUP=kvm
+	img=$(readlink -f /usr/share/kata-containers/kata-containers.img)
+	sudo chown -R "root:$TRUSTED_GROUP" "$img"
+	sudo chmod -R g+rw "$img"
+}
+
+setup_podman_rootless() {
+	[ -e ~/.config/containers/libpod.conf ] || install -D /usr/share/containers/libpod.conf ~/.config/containers/libpod.conf
+	sed -i -e "s|^tmp_dir = .*$|tmp_dir = \"$XDG_RUNTIME_DIR/libpod/tmp\"|" ~/.config/containers/libpod.conf
+}
+
+kata_runtime_podman() {
+	sed -i -e 's/^runtime = "runc"/runtime = "kata"/' ~/.config/containers/libpod.conf
```

Comment:
Aren't you missing the step where you define what the "`kata`" runtime is?

```
$ echo 'kata = ["/usr/bin/kata-runtime"]' >> ~/.config/containers/libpod.conf
```

Taken from the pending podman docs PR (https://github.com/kata-containers/documentation/pull/565).